packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -78,27 +80,61 @@ export const styles = theme => {
         top: 'initial',
         bottom: 0,
         width: 2,
+        height: '100%',
       },
     },
     /* Styles applied to the track element before the thumb. */
     trackBefore: {
       zIndex: 1,
       left: 0,
       transition: trackTransitions,
+      transformOrigin: '0 100%',
+      '&$rtl': {
+        transformOrigin: '0% 100%',
+      },
     },
     /* Styles applied to the track element after the thumb. */
     trackAfter: {
       right: 0,
       opacity: 0.24,
       transition: trackTransitions,
+      transformOrigin: '100% 0',
       '&$vertical': {
-        bottom: 0,
+        top: 0,
+      },
+    },
+    /* Styles applied to the thumb wrapper element. */
+    thumbWrapper: {
+      position: 'relative',
+      zIndex: 2,
+      pointerEvents: 'none',
+      '&$activated': {
+        willChange: 'transform',
+      },
+      '&$rtl': {
+        position: 'relative',
+        left: '100%',
+      },
+      '&$vertical': {
+        position: 'relative',
+        bottom: 12,
+        height: '100%',
+      },
+      '&$vertical$disabled': {
+        bottom: 9,
+      },
+      '&$vertical$rtl': {
```

Comment:
Preferably a vertical slider should not care about RLT behavior. It seems like it's not symmetric anymore? Why is this conditional styling required?

Reply:
The vertical slider is not symmetric right now, see https://material-ui.com/lab/slider/ in RTL mode